gto-support-db/build.gradle

```diff
@@ -14,4 +14,5 @@ dependencies {
     compile project(':gto-support-core')
 
     testCompile project(':gto-support-testing')
+    androidTestCompile 'org.hamcrest:hamcrest-library:1.3'
```

Comment:
I actually ran into the 65k limit on my local machine/branch pulling in test dependencies. Do I need to set up some sort of special thing to avoid that?